chromium/chromium.dockerfile

```diff
@@ -1,20 +1,26 @@
 FROM janx/ubuntu-dev
-MAINTAINER Jan Keromnes "janx@linux.com"
 
 # Install Chromium's depot_tools.
 RUN git clone https://chromium.googlesource.com/chromium/tools/depot_tools.git
 ENV PATH $PATH:/home/user/depot_tools
-RUN echo "\n# Add Chromium's depot_tools to the PATH." >> .bashrc \
- && echo "export PATH=\"\$PATH:/home/user/depot_tools\"" >> .bashrc
+ENV GYP_CHROMIUM_NO_ACTION 1
+RUN cat << EOD >> .bashrc
```

Comment:
Nit: While we're at it, we might as well use the absolute path `/home/user/.bashrc`.

Reply:
Done.